content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md

```diff
@@ -1824,7 +1824,7 @@ PodStatus represents information about the status of a pod. Status may trail the
 
 - **phase** (string)
 
-  The phase of a Pod is a simple, high-level summary of where the Pod is in its lifecycle. The conditions array, the reason and message fields, and the individual container status arrays contain more detail about the pod's status. There are five possible phase values:
+  The phase of a Pod is a high-level summary of where the Pod is in its lifecycle. The conditions array, the reason and message fields, and the individual container status arrays contain more detail about the pod's status. There are five possible phase values:
```

Comment:
I don't think the word "simple" here should be removed.

Reply:
I don't think "simple" is adding anything useful here, but ymmv 🤷🏼‍♀️

Reply:
This file was auto-generated from the upstream source code. Please fix it in the kubernetes/kubernetes repo.

Reply:
Removing “simple” from the API definition is OK by me. It's also OK to _file an issue_ about removing that word (with some justification).